src/applications/dhp-connected-devices/containers/App.jsx

```diff
@@ -21,43 +26,11 @@ export default function App() {
             your VA care team. If you have concerns about any specific shared
             data, you must contact your care team directly.
           </p>
-          <div className="schemaform-title">
-            <h2>Connected devices</h2>
-          </div>
-          <va-alert
-            close-btn-aria-label="Close notification"
-            status="continue"
-            visible
-          >
-            <h3 slot="headline">Please sign in to connect a device</h3>
-            <div>
-              Sign in with your existing ID.me, DS Logon, or My HealtheVet
-              account. If you don't have any of these accounts, you can create a
-              free ID.me account now.
-            </div>
-            <div>
-              <button type="button" className="usa-button">
-                Sign in or create an account
-              </button>
-            </div>
-          </va-alert>
-          <div className="schemaform-title">
-            <h2>Frequently asked questions</h2>
-          </div>
-          <va-accordion
-            disable-analytics={{
-              value: 'false',
-            }}
-            section-heading={{
-              value: 'null',
-            }}
-          >
-            <va-accordion-item id="first">
-              <h6 slot="headline">Question 1</h6>
-            </va-accordion-item>
-            <va-accordion-item header="Question 2" id="second" />
-            <va-accordion-item header="Question 3" id="third" />
-          </va-accordion>
+          {/* Show sign in button if user not logged in */}
+          {!isLoggedIn && <UnauthenticatedPageContent />}
+          {/* show your devices and Connect device section if user is logged in */}
+          {isLoggedIn && <AuthenticatedPageContent />}
```

Comment:
Loving this refactor ❤️

Reply:
One minor nitpick - Code comments are probably not needed here and usually become outdated quickly